packages/agoric-cli/src/commands/psm.js

```diff
@@ -182,6 +217,115 @@ export const makePsmCommand = async logger => {
         method: 'executeOffer',
         offer,
       });
+
+      console.warn('Now execute the prepared offer');
+    });
+
+  psm
+    .command('propose')
+    .description('propose a vote (hard-coded for VoteOnPauseOffers so far)')
+    .option('--offerId [number]', 'Offer id', Number, Date.now())
+    .requiredOption(
+      '--previousOfferId [number]',
+      'offer that had continuing invitation result',
+      Number,
+    )
+    .option(
+      '--invitationMaker [name]',
+      'invitationMakerName',
+      'VoteOnPauseOffers',
+    )
+    .action(async function () {
+      const opts = this.opts();
+
+      const psmInstance = agoricNames.instance['psm-IST-AUSD'];
+      assert(psmInstance, 'missing psmInstance');
+
+      /** @type {import('../lib/psm.js').OfferSpec} */
+      const offer = {
+        id: Number(opts.offerId),
+        invitationSpec: {
+          source: 'continuing',
+          previousOffer: opts.previousOfferId,
+          invitationMakerName: opts.invitationMaker,
+          // instance, strings list, timer deadline (seconds)
+          invitationArgs: harden([
+            psmInstance,
+            ['wantMinted'],
```

Comment:
Also eventually a parameter

Reply:
I forgot to parameterize this. In practice we can edit the source so I don't want to make every little thing a parameter, especially for PS0.

Reply:
Agreed.  You were commenting some of those to be parameterized, so I was hoping for similar treatment here.

Reply:
Since I was pushing again anyway I parameterized this